src/fps/task/MultisigTask.sol

```diff
@@ -200,6 +178,16 @@ abstract contract MultisigTask is Test, Script, ITask {
         /// check that the safe address is the same for all chains and then set safe in storage
         multisig = addresses.getAddress(config.safeAddressString, chains[0].chainId);
 
+        /// TODO change this once we implement task stacking
+        nonce = IGnosisSafe(multisig).nonce();
+
+        address[] memory owners = IGnosisSafe(multisig).getOwners();
+        for (uint256 i = 0; i < owners.length; i++) {
```

Comment:
I think we want this logic to be what Matt mentioned on discord: https://discord.com/channels/1244729134312198194/1328413719402647576/1332347425561641021

i.e. If there are _any_ EOAs then we can call it a single multisig.

Reply:
awesome, resolving this issue.